client/verta/tests/test_entities.py

```diff
@@ -174,6 +174,23 @@ def config_file_with_type_util(self, connect):
         finally:
             os.environ[HOST_KEY], os.environ[EMAIL_KEY], os.environ[DEV_KEY_KEY] = HOST, EMAIL, DEV_KEY
 
+    def test_wrong_credentials(self):
```

Comment:
I think this needs to be marked with `@not_oss` (if I even remember the marker name right)

Reply:
fixed.